web/documentserver-example/nodejs/helpers/docManager.js

```diff
@@ -369,4 +369,51 @@ docManager.cleanFolderRecursive = function (folder, me) {
     }
 };
 
+docManager.getFilesInFolderInfo = function (){
+    //new logic
+    const userAddress = docManager.curUserHostAddress();
+    const directory = path.join(docManager.dir, userAddress);
+    const filesInFolder = this.getStoredFiles();
+    var responseArray = [];
+    filesInFolder.forEach((file)=>{
+        const stats = fileSystem.lstatSync(path.join(directory, file.name));
+        const fileObject = {
+            version: file.version,
+            fileStatus: Number(!file.canEdit), //0, if the file can be edited, and 1 if not
```

Comment:
What is the parameter `fileStatus` and why is it numerical?

Reply:
fileStatus is incorrect. I will need to fix this.